### ISSUE TITLE
Inline Keyboard Button

### ISSUE THREAD

**Issue by paduhq** [rehmanali007/telegram-msg-forward#1]
Hello, can you add so that the bot supports to clone inline keyboard buttons too instead of just texts and media ?

**Comment by runetech0** [comment]
@paduhq , I don't think that is possible because to send buttons in the messages you will need to login as a bot account and this script uses user account to login.
